app/models/role.rb

```diff
@@ -4,10 +4,10 @@ class Role < ActiveHash::Base
   include ActiveHash::Enum
 
   self.data = [
-    { id: 0, name: "viewer" },
-    { id: 1, name: "deployer" },
-    { id: 2, name: "admin" },
-    { id: 3, name: "super_admin" }
+    { id: 0, name: "viewer", display_name: "Viewer" },
+    { id: 1, name: "deployer", display_name: "Deployer" },
+    { id: 2, name: "admin", display_name: "Admin" },
+    { id: 3, name: "super_admin", display_name: "Super Admin" }
```

Comment:
Damn, didn't find that function. I did search for capitalize, but then noticed the underscores and gave up.

I'll remove the `display_name` property and create a method with the same name, returning `name.humanize`. I won't have to change it in all the places but it will make the hash cleaner.

Thanks.